lxd/cluster/membership.go

```diff
@@ -829,13 +829,13 @@ func Assign(state *state.State, gateway *Gateway, nodes []db.RaftNode) error {
 		defer func() { _ = client.Close() }()
 
 		// Figure out our current role.
-		role := db.RaftRole(-1)
-		cluster, err := client.Cluster(ctx)
+		role := db.RaftRole(-1) // -1 is an invalid role.
+		servers, err := client.Cluster(ctx)
```

Comment:
`members` would have been more inline with our existing terminology. But `servers` is certainly still clearer than `cluster`, so happy to keep as-is.